api/schemas/organization.yaml

```diff
@@ -0,0 +1,35 @@
+type: object
+description: Organization's data
+properties:
+  id:
+    type: integer
+    format: uint32
+  name:
+    type: string
+  description:
+    type: string
+  # reference to LDAP configuration
+  ldapConfiguration:
+    $ref: "ldapConfiguration.yaml"
+    description: URL or IP address of the LDAP server of the organization. Useful only if the organization is private.
+  # reference to Place
+  places:
+    type: array
+    description: list of places within this organization
+    items:
+      $ref: "place.yaml"
+  # if @private == true a user can't be anonymous
+  private:
```

Comment:
private è una keyword in angular, crea problemi

Reply:
Da modificare in _isPrIvate_